xml/System.Linq/Enumerable.xml

```diff
@@ -11273,9 +11273,7 @@ If `count` is not a positive number, this method returns an identical copy of th
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- This method returns zero if `source` contains no elements.
-
- The result does not include values that are `null`.
+ Items in `source` that are `null` are excluded from the computation of the sum. This method returns zero if `source` contains no elements or all elements are `null`.
```

Comment:
Since you are changing all these remarks, could you also add a blank line after the heading:

```suggestion

Items in `source` that are `null` are excluded from the computation of the sum. This method returns zero if `source` contains no elements or all elements are `null`.
```

Reply:
updared

Reply:
@gewarren 
done applying feedback on `Sum()`. Please review.